src/util/db.rs

```diff
@@ -155,6 +155,13 @@ macro_rules! impl_byte_array_from_column {
                 Ok(inst)
             }
         }
+
+        impl rusqlite::types::ToSql for $thing {
```

Comment:
Ah, if we're going to do this, then we should probably go and remove all the calls to `$thing.to_hex() as &dyn ToSql` in the database code.

Reply:
Okay, will do

Reply:
Updated in 7a9fb081c1ed9d76003793c3b39c2e08bc14c3aa -- judging by the vast majority of the remaining `.to_hex()` usage, do you think we should just `impl Fmt::Display` for these types as well ?

Reply:
Thanks for doing that!  I'm fine with doing this for `fmt::Display` as well, if you're up for it.

Reply:
Yep, will add Fmt::Display for this as well.

Reply:
Updated in 566f1be5d341032eddd9309bb41c317af2d9d0f0